articles/agora-express-token-server/index.md

```diff
@@ -0,0 +1,364 @@
+
+![Agora Project Management Console App ID](project_management_console_app_id.png)
+
+Now, click on the Edit action.
+
+![Agora Project Management Console Edit Button](project_management_console_edit.png)
+
+This will take you to the Edit Project screen. Here, you can copy the primary certificate which is required to generate the tokens.
+
+![Primary Certificate](primary_certificate.png)
+
+> The app ID and the app certificates must be kept secret. If the primary certificate gets compromised, you can enable a secondary app certificate and replace the current certificate with the secondary certificate. Later, you can delete the compromised app certificate. Make sure you update the token server with the new app certificate.
+
+If you've set the Authentication Mode to Testing while creating the project, the primary certificate will be disabled. You can enable the primary certificate by clicking on Enable.
+
+![Disabled Primary Certificate](disabled_primary_certificate.png)
+
+### Generating a Temporary Token
+
+For testing purposes, you can generate a temporary token from the project management console. Click on the temporary token button next to the app ID in the project management console.
+
+![Agora Project Management Console Temp Token](project_management_console_temp_token.png)
+
+This will take you to generate a temporary token form. You need to fill in the channel name and then click on the Generate Temp token button. You'll get a temporary token which you can use in your app for testing.
+
+![Agora Project Management Console Temp Token](temp_token_form.png)
+
+### Setting up the Server
+> You need Node.js to set up an Express server. You can download Node.js from [here](https://nodejs.org/en/).
+
+Let's install `Express` using `NPM`.
+
+```bash
+npm install express
+```
+
+Now, we can import `express` in our code to create a simple server module that'll listen on port 3000.
+
+```JavaScript
+const express = require("express");
+const app = express();
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("Agora Auth Token Server"));
+
+const port = process.env.PORT || 3000;
+app.listen(port, () => console.log(`Agora Auth Token Server listening at Port ${port}`));
+```
+
+You can start the server by running:
+
+```bash
+node index.js
+```
+
+This server will be listening on port 3000 and when you hit the `'/'` endpoint, it'll send `"Agora Auth Token Server"`.
+
+### Generating authentication Tokens
+
+Agora provides the RTC SDK and the RTM SDK. The `agora-access-token` package is used to generate tokens for both the SDKs.
+
+The RTC (Real-time Communication) SDK is used to enable real-time audio and video communications. With the help of the RTC SDK, developers can add audio/video call and audio/video broadcast functions in their projects. You can learn more about the RTC SDK [here](https://docs.agora.io/en/Agora%20Platform/term_agora_rtc_sdk?platform=Android).
```

Comment:
Once you have mentioned this full name when you first introduce RTC you won't need to point it out here

Reply:
Done.